pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -180,7 +182,9 @@ type ResourceSpec struct {
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// ServiceAffinityArgs holds arguments used to configure the ServiceAffinity plugin.
+// ServiceAffinityArgs holds arguments used to configure the ServiceAffinity
+// plugin. Note: this plugin has been deprecated and is only configurable
```

Comment:
nit: https://blog.golang.org/godoc#:~:text=Deprecated

Reply:
so I reverted this change because of staticcheck, which checks for uses of symbols that have "Deprecated:" in their docs. Since we still use these types in a lot of places in the scheduler code, I think it makes sense to back to not using "Deprecated:". WDYT?